tools/journal_proto_processor.hpp

```diff
@@ -149,6 +149,18 @@ class JournalProtoProcessor {
                                      std::string const& expr2)>>
       field_cxx_inserter_fn_;
 
+  // For all fields, a lambda that takes a C#/C++ parameter type as stored in
+  // |field_cs_type_|, repsectively |field_cxx_type_|, and adds a mode to it,
+  // yielding |T| for an in parameter in either language, |ref T| for an in out
```

Comment:
I'd drop everything from "yielding" to the end, no point in leaking implementation details.